[13-01-2020 17:45:33];
        Текст ошибки: Uncaught TypeError: Argument 1 passed to myframe\core\base\ErrorHandler::exceptionHandler() must be an instance of Exception, instance of Error given in /var/www/myframe.com/vendor/myframe/core/base/ErrorHandler.php:29
Stack trace:
#0 [internal function]: myframe\core\base\ErrorHandler->exceptionHandler(Object(Error))
#1 {main}
  thrown;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/base/ErrorHandler.php;
        Строка ошибки: 29;
        //============================================//

[13-01-2020 17:46:30];
        Текст ошибки: Uncaught TypeError: Argument 1 passed to myframe\core\base\ErrorHandler::exceptionHandler() must be an instance of Exception, instance of Error given in /var/www/myframe.com/vendor/myframe/core/base/ErrorHandler.php:29
Stack trace:
#0 [internal function]: myframe\core\base\ErrorHandler->exceptionHandler(Object(Error))
#1 {main}
  thrown;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/base/ErrorHandler.php;
        Строка ошибки: 29;
        //============================================//

[13-01-2020 17:46:31];
        Текст ошибки: Uncaught TypeError: Argument 1 passed to myframe\core\base\ErrorHandler::exceptionHandler() must be an instance of Exception, instance of Error given in /var/www/myframe.com/vendor/myframe/core/base/ErrorHandler.php:29
Stack trace:
#0 [internal function]: myframe\core\base\ErrorHandler->exceptionHandler(Object(Error))
#1 {main}
  thrown;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/base/ErrorHandler.php;
        Строка ошибки: 29;
        //============================================//

[13-01-2020 17:46:31];
        Текст ошибки: Uncaught TypeError: Argument 1 passed to myframe\core\base\ErrorHandler::exceptionHandler() must be an instance of Exception, instance of Error given in /var/www/myframe.com/vendor/myframe/core/base/ErrorHandler.php:29
Stack trace:
#0 [internal function]: myframe\core\base\ErrorHandler->exceptionHandler(Object(Error))
#1 {main}
  thrown;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/base/ErrorHandler.php;
        Строка ошибки: 29;
        //============================================//

[13-01-2020 17:49:01];
        Текст ошибки: asdas;
Файл ошибки: /var/www/myframe.com/app/controllers/MainController.php;
        Строка ошибки: 16;
        //============================================//

[13-01-2020 17:49:18];
        Текст ошибки: Undefined variable: sda;
Файл ошибки: /var/www/myframe.com/app/controllers/MainController.php;
        Строка ошибки: 16;
        //============================================//

[13-01-2020 17:49:47];
        Текст ошибки: Uncaught TypeError: Argument 1 passed to myframe\core\base\ErrorHandler::exceptionHandler() must be an instance of Exception, instance of Error given in /var/www/myframe.com/vendor/myframe/core/base/ErrorHandler.php:29
Stack trace:
#0 [internal function]: myframe\core\base\ErrorHandler->exceptionHandler(Object(Error))
#1 {main}
  thrown;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/base/ErrorHandler.php;
        Строка ошибки: 29;
        //============================================//

[13-01-2020 17:50:27];
        Текст ошибки: Uncaught Error: Call to undefined function app\controllers\test() in /var/www/myframe.com/app/controllers/MainController.php:16
Stack trace:
#0 /var/www/myframe.com/vendor/myframe/core/Router.php(121): app\controllers\MainController->actionIndex()
#1 /var/www/myframe.com/vendor/myframe/core/App.php(59): myframe\core\Router->dispatch()
#2 /var/www/myframe.com/web/index.php(31): myframe\core\App->run()
#3 {main}
  thrown;
Файл ошибки: /var/www/myframe.com/app/controllers/MainController.php;
        Строка ошибки: 16;
        //============================================//

[13-01-2020 17:50:45];
        Текст ошибки: Uncaught Error: Call to undefined function app\controllers\test() in /var/www/myframe.com/app/controllers/MainController.php:16
Stack trace:
#0 /var/www/myframe.com/vendor/myframe/core/Router.php(121): app\controllers\MainController->actionIndex()
#1 /var/www/myframe.com/vendor/myframe/core/App.php(59): myframe\core\Router->dispatch()
#2 /var/www/myframe.com/web/index.php(31): myframe\core\App->run()
#3 {main}
  thrown;
Файл ошибки: /var/www/myframe.com/app/controllers/MainController.php;
        Строка ошибки: 16;
        //============================================//

[13-01-2020 17:50:46];
        Текст ошибки: Uncaught Error: Call to undefined function app\controllers\test() in /var/www/myframe.com/app/controllers/MainController.php:16
Stack trace:
#0 /var/www/myframe.com/vendor/myframe/core/Router.php(121): app\controllers\MainController->actionIndex()
#1 /var/www/myframe.com/vendor/myframe/core/App.php(59): myframe\core\Router->dispatch()
#2 /var/www/myframe.com/web/index.php(31): myframe\core\App->run()
#3 {main}
  thrown;
Файл ошибки: /var/www/myframe.com/app/controllers/MainController.php;
        Строка ошибки: 16;
        //============================================//

[13-01-2020 17:50:46];
        Текст ошибки: Uncaught Error: Call to undefined function app\controllers\test() in /var/www/myframe.com/app/controllers/MainController.php:16
Stack trace:
#0 /var/www/myframe.com/vendor/myframe/core/Router.php(121): app\controllers\MainController->actionIndex()
#1 /var/www/myframe.com/vendor/myframe/core/App.php(59): myframe\core\Router->dispatch()
#2 /var/www/myframe.com/web/index.php(31): myframe\core\App->run()
#3 {main}
  thrown;
Файл ошибки: /var/www/myframe.com/app/controllers/MainController.php;
        Строка ошибки: 16;
        //============================================//

[13-01-2020 17:50:47];
        Текст ошибки: Uncaught Error: Call to undefined function app\controllers\test() in /var/www/myframe.com/app/controllers/MainController.php:16
Stack trace:
#0 /var/www/myframe.com/vendor/myframe/core/Router.php(121): app\controllers\MainController->actionIndex()
#1 /var/www/myframe.com/vendor/myframe/core/App.php(59): myframe\core\Router->dispatch()
#2 /var/www/myframe.com/web/index.php(31): myframe\core\App->run()
#3 {main}
  thrown;
Файл ошибки: /var/www/myframe.com/app/controllers/MainController.php;
        Строка ошибки: 16;
        //============================================//

[13-01-2020 17:50:47];
        Текст ошибки: Uncaught Error: Call to undefined function app\controllers\test() in /var/www/myframe.com/app/controllers/MainController.php:16
Stack trace:
#0 /var/www/myframe.com/vendor/myframe/core/Router.php(121): app\controllers\MainController->actionIndex()
#1 /var/www/myframe.com/vendor/myframe/core/App.php(59): myframe\core\Router->dispatch()
#2 /var/www/myframe.com/web/index.php(31): myframe\core\App->run()
#3 {main}
  thrown;
Файл ошибки: /var/www/myframe.com/app/controllers/MainController.php;
        Строка ошибки: 16;
        //============================================//

[13-01-2020 17:51:11];
        Текст ошибки: Uncaught Error: Call to undefined function app\controllers\test() in /var/www/myframe.com/app/controllers/MainController.php:16
Stack trace:
#0 /var/www/myframe.com/vendor/myframe/core/Router.php(121): app\controllers\MainController->actionIndex()
#1 /var/www/myframe.com/vendor/myframe/core/App.php(59): myframe\core\Router->dispatch()
#2 /var/www/myframe.com/web/index.php(31): myframe\core\App->run()
#3 {main}
  thrown;
Файл ошибки: /var/www/myframe.com/app/controllers/MainController.php;
        Строка ошибки: 16;
        //============================================//

[13-01-2020 17:51:13];
        Текст ошибки: Uncaught Error: Call to undefined function app\controllers\test() in /var/www/myframe.com/app/controllers/MainController.php:16
Stack trace:
#0 /var/www/myframe.com/vendor/myframe/core/Router.php(121): app\controllers\MainController->actionIndex()
#1 /var/www/myframe.com/vendor/myframe/core/App.php(59): myframe\core\Router->dispatch()
#2 /var/www/myframe.com/web/index.php(31): myframe\core\App->run()
#3 {main}
  thrown;
Файл ошибки: /var/www/myframe.com/app/controllers/MainController.php;
        Строка ошибки: 16;
        //============================================//

[13-01-2020 17:51:51];
        Текст ошибки: Uncaught Exception: asdas in /var/www/myframe.com/app/controllers/MainController.php:16
Stack trace:
#0 /var/www/myframe.com/vendor/myframe/core/Router.php(121): app\controllers\MainController->actionIndex()
#1 /var/www/myframe.com/vendor/myframe/core/App.php(59): myframe\core\Router->dispatch()
#2 /var/www/myframe.com/web/index.php(31): myframe\core\App->run()
#3 {main}
  thrown;
Файл ошибки: /var/www/myframe.com/app/controllers/MainController.php;
        Строка ошибки: 16;
        //============================================//

[13-01-2020 17:52:21];
        Текст ошибки: asdas;
Файл ошибки: /var/www/myframe.com/app/controllers/MainController.php;
        Строка ошибки: 16;
        //============================================//

[13-01-2020 17:53:06];
        Текст ошибки: Uncaught TypeError: Argument 1 passed to myframe\core\base\ErrorHandler::exceptionHandler() must be an instance of Exception, instance of Error given in /var/www/myframe.com/vendor/myframe/core/base/ErrorHandler.php:29
Stack trace:
#0 [internal function]: myframe\core\base\ErrorHandler->exceptionHandler(Object(Error))
#1 {main}
  thrown;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/base/ErrorHandler.php;
        Строка ошибки: 29;
        //============================================//

[13-01-2020 17:54:03];
        Текст ошибки: Uncaught TypeError: Argument 1 passed to myframe\core\base\ErrorHandler::exceptionHandler() must be an instance of Exception, instance of Error given in /var/www/myframe.com/vendor/myframe/core/base/ErrorHandler.php:29
Stack trace:
#0 [internal function]: myframe\core\base\ErrorHandler->exceptionHandler(Object(Error))
#1 {main}
  thrown;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/base/ErrorHandler.php;
        Строка ошибки: 29;
        //============================================//

[13-01-2020 17:54:04];
        Текст ошибки: Uncaught TypeError: Argument 1 passed to myframe\core\base\ErrorHandler::exceptionHandler() must be an instance of Exception, instance of Error given in /var/www/myframe.com/vendor/myframe/core/base/ErrorHandler.php:29
Stack trace:
#0 [internal function]: myframe\core\base\ErrorHandler->exceptionHandler(Object(Error))
#1 {main}
  thrown;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/base/ErrorHandler.php;
        Строка ошибки: 29;
        //============================================//

[13-01-2020 17:54:04];
        Текст ошибки: Uncaught TypeError: Argument 1 passed to myframe\core\base\ErrorHandler::exceptionHandler() must be an instance of Exception, instance of Error given in /var/www/myframe.com/vendor/myframe/core/base/ErrorHandler.php:29
Stack trace:
#0 [internal function]: myframe\core\base\ErrorHandler->exceptionHandler(Object(Error))
#1 {main}
  thrown;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/base/ErrorHandler.php;
        Строка ошибки: 29;
        //============================================//

[13-01-2020 17:54:24];
        Текст ошибки: Uncaught TypeError: Argument 1 passed to myframe\core\base\ErrorHandler::exceptionHandler() must be an instance of Exception, instance of Error given in /var/www/myframe.com/vendor/myframe/core/base/ErrorHandler.php:29
Stack trace:
#0 [internal function]: myframe\core\base\ErrorHandler->exceptionHandler(Object(Error))
#1 {main}
  thrown;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/base/ErrorHandler.php;
        Строка ошибки: 29;
        //============================================//

[13-01-2020 17:54:25];
        Текст ошибки: Uncaught TypeError: Argument 1 passed to myframe\core\base\ErrorHandler::exceptionHandler() must be an instance of Exception, instance of Error given in /var/www/myframe.com/vendor/myframe/core/base/ErrorHandler.php:29
Stack trace:
#0 [internal function]: myframe\core\base\ErrorHandler->exceptionHandler(Object(Error))
#1 {main}
  thrown;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/base/ErrorHandler.php;
        Строка ошибки: 29;
        //============================================//

[13-01-2020 17:54:25];
        Текст ошибки: Uncaught TypeError: Argument 1 passed to myframe\core\base\ErrorHandler::exceptionHandler() must be an instance of Exception, instance of Error given in /var/www/myframe.com/vendor/myframe/core/base/ErrorHandler.php:29
Stack trace:
#0 [internal function]: myframe\core\base\ErrorHandler->exceptionHandler(Object(Error))
#1 {main}
  thrown;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/base/ErrorHandler.php;
        Строка ошибки: 29;
        //============================================//

[13-01-2020 17:55:02];
        Текст ошибки: Uncaught TypeError: Argument 1 passed to myframe\core\base\ErrorHandler::exceptionHandler() must be an instance of Exception, instance of Error given in /var/www/myframe.com/vendor/myframe/core/base/ErrorHandler.php:29
Stack trace:
#0 [internal function]: myframe\core\base\ErrorHandler->exceptionHandler(Object(Error))
#1 {main}
  thrown;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/base/ErrorHandler.php;
        Строка ошибки: 29;
        //============================================//

[13-01-2020 17:55:02];
        Текст ошибки: Uncaught TypeError: Argument 1 passed to myframe\core\base\ErrorHandler::exceptionHandler() must be an instance of Exception, instance of Error given in /var/www/myframe.com/vendor/myframe/core/base/ErrorHandler.php:29
Stack trace:
#0 [internal function]: myframe\core\base\ErrorHandler->exceptionHandler(Object(Error))
#1 {main}
  thrown;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/base/ErrorHandler.php;
        Строка ошибки: 29;
        //============================================//

[13-01-2020 17:55:03];
        Текст ошибки: Uncaught TypeError: Argument 1 passed to myframe\core\base\ErrorHandler::exceptionHandler() must be an instance of Exception, instance of Error given in /var/www/myframe.com/vendor/myframe/core/base/ErrorHandler.php:29
Stack trace:
#0 [internal function]: myframe\core\base\ErrorHandler->exceptionHandler(Object(Error))
#1 {main}
  thrown;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/base/ErrorHandler.php;
        Строка ошибки: 29;
        //============================================//

[13-01-2020 17:55:03];
        Текст ошибки: Uncaught TypeError: Argument 1 passed to myframe\core\base\ErrorHandler::exceptionHandler() must be an instance of Exception, instance of Error given in /var/www/myframe.com/vendor/myframe/core/base/ErrorHandler.php:29
Stack trace:
#0 [internal function]: myframe\core\base\ErrorHandler->exceptionHandler(Object(Error))
#1 {main}
  thrown;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/base/ErrorHandler.php;
        Строка ошибки: 29;
        //============================================//

[13-01-2020 17:55:23];
        Текст ошибки: Call to undefined function app\controllers\test();
Файл ошибки: /var/www/myframe.com/app/controllers/MainController.php;
        Строка ошибки: 16;
        //============================================//

[13-01-2020 18:02:14];
        Текст ошибки: Страница не найдена;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 131;
        //============================================//

[13-01-2020 18:02:20];
        Текст ошибки: Kласс app\controllers\DasdController не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 128;
        //============================================//

[13-01-2020 18:02:41];
        Текст ошибки: Kласс app\controllers\DasdController не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 128;
        //============================================//

[13-01-2020 18:02:47];
        Текст ошибки: Метод actionAsdsa не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 125;
        //============================================//

[13-01-2020 18:03:16];
        Текст ошибки: Метод actionAsdsa не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 125;
        //============================================//

[13-01-2020 18:03:33];
        Текст ошибки: Метод actionAsdsa не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 125;
        //============================================//

[13-01-2020 18:03:44];
        Текст ошибки: Метод actionAsdsa не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 125;
        //============================================//

[13-01-2020 18:03:49];
        Текст ошибки: Метод actionAsdsa не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 125;
        //============================================//

[14-01-2020 17:38:47];
        Текст ошибки: Страница не найдена;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 133;
        //============================================//

[14-01-2020 17:40:44];
        Текст ошибки: Kласс app\controllers\TestController не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 130;
        //============================================//

[14-01-2020 17:42:10];
        Текст ошибки: Kласс app\controllers\SersController не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 130;
        //============================================//

[14-01-2020 17:42:46];
        Текст ошибки: Kласс app\controllers\SersController не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 130;
        //============================================//

[14-01-2020 17:42:53];
        Текст ошибки: Kласс app\controllers\SersController не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 130;
        //============================================//

[14-01-2020 17:42:54];
        Текст ошибки: Kласс app\controllers\SersController не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 130;
        //============================================//

[14-01-2020 17:43:02];
        Текст ошибки: Kласс app\controllers\SersController не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 130;
        //============================================//

[14-01-2020 17:44:20];
        Текст ошибки: Страница не найдена;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 133;
        //============================================//

[14-01-2020 17:44:25];
        Текст ошибки: Kласс app\controllers\SersController не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 130;
        //============================================//

[14-01-2020 17:53:12];
        Текст ошибки: Kласс app\controllers\DsadController не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 130;
        //============================================//

[14-01-2020 17:57:48];
        Текст ошибки: Kласс app\controllers\DsadController не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 136;
        //============================================//

[14-01-2020 20:33:11];
        Текст ошибки: Kласс app\controllers\UserController не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 136;
        //============================================//

[14-01-2020 20:34:14];
        Текст ошибки: Kласс app\controllers\admin\UserController не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 137;
        //============================================//

[14-01-2020 21:07:34];
        Текст ошибки: Kласс app\controllers\admin\UserController не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 137;
        //============================================//

[14-01-2020 21:10:46];
        Текст ошибки: Kласс app\controllers\admin-panel\UserController не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 137;
        //============================================//

[14-01-2020 21:11:11];
        Текст ошибки: Kласс app\controllers\admin-panel\UserController не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 137;
        //============================================//

[14-01-2020 21:11:12];
        Текст ошибки: Kласс app\controllers\admin-panel\UserController не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 137;
        //============================================//

[14-01-2020 21:11:12];
        Текст ошибки: Kласс app\controllers\admin-panel\UserController не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 137;
        //============================================//

[14-01-2020 21:12:47];
        Текст ошибки: Файл вида /var/www/myframe.com/app/views/user/index.php не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/base/View.php;
        Строка ошибки: 100;
        //============================================//

[14-01-2020 21:14:10];
        Текст ошибки: Файл вида /var/www/myframe.com/app/views/user/index.php не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/base/View.php;
        Строка ошибки: 100;
        //============================================//

[14-01-2020 21:17:24];
        Текст ошибки: Kласс app\controllers\adminpanel\UserController не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 137;
        //============================================//

[14-01-2020 21:17:27];
        Текст ошибки: Kласс app\controllers\adminpanel\UserController не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 137;
        //============================================//

[14-01-2020 21:18:38];
        Текст ошибки: Файл вида /var/www/myframe.com/app/views/user/index.php не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/base/View.php;
        Строка ошибки: 100;
        //============================================//

[14-01-2020 21:19:03];
        Текст ошибки: Файл вида /var/www/myframe.com/app/views/user/index.php не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/base/View.php;
        Строка ошибки: 100;
        //============================================//

[14-01-2020 21:27:12];
        Текст ошибки: Файл вида /var/www/myframe.com/app/views/adminpanel\user/index.php не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/base/View.php;
        Строка ошибки: 109;
        //============================================//

[14-01-2020 21:28:49];
        Текст ошибки: Файл вида /var/www/myframe.com/app/views/adminpanel\user/index.php не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/base/View.php;
        Строка ошибки: 114;
        //============================================//

[14-01-2020 21:28:49];
        Текст ошибки: Файл вида /var/www/myframe.com/app/views/adminpanel\user/index.php не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/base/View.php;
        Строка ошибки: 114;
        //============================================//

[14-01-2020 21:28:50];
        Текст ошибки: Файл вида /var/www/myframe.com/app/views/adminpanel\user/index.php не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/base/View.php;
        Строка ошибки: 114;
        //============================================//

[14-01-2020 21:28:50];
        Текст ошибки: Файл вида /var/www/myframe.com/app/views/adminpanel\user/index.php не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/base/View.php;
        Строка ошибки: 114;
        //============================================//

[14-01-2020 21:30:29];
        Текст ошибки: Файл вида /var/www/myframe.com/app/views/adminpanel\user/index.php не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/base/View.php;
        Строка ошибки: 114;
        //============================================//

[14-01-2020 21:30:29];
        Текст ошибки: Файл вида /var/www/myframe.com/app/views/adminpanel\user/index.php не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/base/View.php;
        Строка ошибки: 114;
        //============================================//

[14-01-2020 21:30:30];
        Текст ошибки: Файл вида /var/www/myframe.com/app/views/adminpanel\user/index.php не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/base/View.php;
        Строка ошибки: 114;
        //============================================//

[14-01-2020 21:31:12];
        Текст ошибки: Файл вида /var/www/myframe.com/app/views/adminpanel\user/index.php не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/base/View.php;
        Строка ошибки: 114;
        //============================================//

[14-01-2020 21:40:09];
        Текст ошибки: Kласс app\controllers\AdminController не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 137;
        //============================================//

[14-01-2020 21:43:14];
        Текст ошибки: Kласс app\controllers\AsdController не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 137;
        //============================================//

[14-01-2020 21:43:38];
        Текст ошибки: Kласс app\controllers\adminpanel\AsdController не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 137;
        //============================================//

[14-01-2020 21:51:46];
        Текст ошибки: Kласс app\controllers\admin\UserController не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 137;
        //============================================//

[14-01-2020 21:52:07];
        Текст ошибки: Kласс app\controllers\admin\UserController не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 137;
        //============================================//

[14-01-2020 21:52:08];
        Текст ошибки: Kласс app\controllers\admin\UserController не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 137;
        //============================================//

[14-01-2020 21:52:08];
        Текст ошибки: Kласс app\controllers\admin\UserController не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 137;
        //============================================//

[14-01-2020 21:52:09];
        Текст ошибки: Kласс app\controllers\admin\UserController не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 137;
        //============================================//

[14-01-2020 21:52:12];
        Текст ошибки: Kласс app\controllers\admin\UserController не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 137;
        //============================================//

[14-01-2020 21:52:12];
        Текст ошибки: Kласс app\controllers\admin\UserController не найден;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/Router.php;
        Строка ошибки: 137;
        //============================================//

[14-01-2020 21:58:03];
        Текст ошибки: Undefined index: controller;
Файл ошибки: /var/www/myframe.com/vendor/myframe/core/base/Controller.php;
        Строка ошибки: 43;
        //============================================//

